gapic-generator/templates/default/service/rest/test/method/_normal.erb

```diff
@@ -13,9 +13,7 @@ def test_<%= method.name %>
   <%= field.name %> = <%= field.default_value %>
   <%- end -%>
 
-  <%= method.name %>_client_stub = ClientStub.new http_response do |verb, uri:, body:, params:, options:|
-    assert_equal :<%= method.rest.verb %>, verb
```

Comment:
We're getting rid of this assertion?

Reply:
yes, the verb can be different between bindings.